sale_margin_delivered/migrations/16.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(env, version):
+    """Store margin delivered precentage as a fraction of 1"""
+    env.cr.execute(
+        """
+        UPDATE sale_order_line
+        SET margin_delivered_percent = margin_delivered_percent / 100
```

Comment:
Should this be rounded?

Reply:
@pedrobaeza This field is a float field without precision so it should not be rounded.

Reply:
But AFAIK, when no rounding is applied, web client applies 2 digits rounding when saving from it. Is this field shown in web client and thus stripped by it?

Reply:
In a quick test in runboat, it seems that at least 4 digits are respected, so this seems OK.

Reply:
rounding is done on the client side by the use of the 'percentage' widget

Reply:
Yeah, my fear is that on new records, a rounding would be also applied server-side, and then, data was not consistent on analysis. I remember some time ago to have this problem (reported by @alexis-via) with `account.move.line` with `credit` and `debit` not rounded to 2 digits, and this causing differences in aggregations.

Reply:
If no precision is specified on the field declaration, values are never rounded on server side. It's required to avoid rounding issues as the sum of rounded values != to the round of the sum of values.

Reply:
I didn't express correctly, sorry. It was not server-side, but the web client passing rounded values to the server. Anyway, this seems clear and tried on a functional test. Thanks for the clarifications.

Reply:
I have a mix of both kind of issues (server side and client side) in my memories, and I want to avoid both.